cmd/harbor/root/labels/list.go

```diff
@@ -15,6 +15,7 @@ func ListLabelCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list",
 		Short: "list labels",
+		Args: cobra.ExactArgs(0),
```

Comment:
please do `cobra.NoArgs`
```suggestion
		Args: cobra.NoArgs,
```

to maintain consistency